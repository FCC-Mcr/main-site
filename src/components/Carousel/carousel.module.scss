@import "../../styles/utility/responds-to.scss";

.carousel {
  padding: 80px 40px 40px;

  // @include responds-to("xs", "max") {
  //   padding-top: 240px;
  // }

  scroll-snap-type: x mandatory;
  overflow-x: scroll;
  -webkit-overflow-scrolling: touch;
  white-space: nowrap;
  height: 400px;
}
