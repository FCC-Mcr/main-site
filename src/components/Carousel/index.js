import React from "react"
import styles from "./carousel.module.scss"

const Index = ({ children }) => {
  return (
    <div className={`${styles.carouselWrapper} full-width`}>
      <div className={`${styles.carousel} full-width`}>{children}</div>
    </div>
  )
}

export default Index
