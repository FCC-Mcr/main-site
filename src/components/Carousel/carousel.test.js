import React from "react"
import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

import Carousel from "./"

describe("Carousel section", () => {
  test("Snapshot", () => {
    const { container } = render(<Carousel />)
    expect(container).toMatchSnapshot()
  })
})
