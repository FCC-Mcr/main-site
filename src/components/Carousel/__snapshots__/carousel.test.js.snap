// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel section Snapshot 1`] = `
<div>
  <div
    class="carouselWrapper full-width"
  >
    <div
      class="carousel full-width"
    />
  </div>
</div>
`;
